src/SkeletalTrapezoidation.cpp

```diff
@@ -343,26 +343,6 @@ void SkeletalTrapezoidation::computeSegmentCellRange(vd_t::cell_type& cell, Poin
             ending_vd_edge = edge;
             check_secondary_edge = false;
         }
-        if (false && check_secondary_edge && edge->is_secondary()
```

Comment:
Do we know why the `false` was added in the first place?
I don't think we should put this back but ... with all those TODO's as well, we should keep these changes in the back of our minds when it's time for the inevitable debugging.

Reply:
Nope. But i'm guessing that it's the equalivant of commenting out the code (so figuring out what works and what doesnt, this check not being needed in the end?)

Reply:
Probably yeah.